src/test/ui/issues/issue-75763.rs

```diff
@@ -0,0 +1,14 @@
+// build-pass
+
+#![allow(incomplete_features)]
+#![feature(const_generics)]
+
+struct Bug<const S: &'static str>;
+    
+fn main() {
+    let b: Bug::<{
+        unsafe {
+            std::mem::transmute::<&[u8], &str>(&[0xC0, 0xC1, 0xF5])
```

Comment:
```suggestion
            // FIXME(const_generics): Decide on how to deal with invalid values as const params
            std::mem::transmute::<&[u8], &str>(&[0xC0, 0xC1, 0xF5])
```
So that we don't forget this if we close the issue